src/Component.js

```diff
@@ -1,27 +1,52 @@
-import React, { useState, useEffect } from "react";
+import React from "react";
 import * as props from "./Component.props";
+import DropDownList from "./components/DropDownList";
+import RadioButtonList from "./components/RadioButtonList"
+import { filterOptions } from "./utils/option-params";
+import styled from "styled-components";
 
-// Build a Response Component as a React Functional Component
-const Component = ({ label, text, logResults, setNextEnabled }) => {
-  const [value, setValue] = useState(text);
-  useEffect(() => {
-    setValue(text);
-    setNextEnabled(true);
-  }, [text, setNextEnabled]);
+// Container to sort out component alignment and size
+const Alignment = styled.div`
+  width: ${x => x.dropDown ? (x.width || "100%") : "fit-content"};
+  float: ${x => (x.alignment == "center") ? "" : x.alignment};
+  margin: auto;
+`;
+
+// Main Component
+const Component = ({
+  logResults, 
+  setNextEnabled,
+  dropDown,
+  width,
+  alignment,
+  textColor,
+  fontSize,
+  fontFamily,
+  ...props
+}) => {
 
-  const handleInput = ({ target }) => {
-    setValue(target.value);
-  };
+  const options = filterOptions(props);
 
-  const handleBlur = e => {
-    e.persist();
-    logResults({ text: e.target.value });
-  };
+  const styles = {
+    textColor,
+    fontSize,
+    fontFamily,
+  }
+
+  const logOption = option => {
+    setNextEnabled(true);
+    logResults(option);
+  }
 
   return (
+    // Parent <div> required for alignment in Decsys
```

Comment:
display flex on Alignment might also resolve the need for this wrapper. but don't quote me on that.